frontend/src/metabase/admin/datamodel/components/FieldRemapping.styled.tsx

```diff
@@ -1,12 +1,28 @@
 import styled from "@emotion/styled";
-import { color } from "metabase/lib/colors";
+import { alpha, color } from "metabase/lib/colors";
+import SelectButton from "metabase/core/components/SelectButton";
 import FieldList from "metabase/query_builder/components/FieldList";
 
+export const FieldMappingRoot = styled.div`
+  padding: 1rem 4rem;
+  border: 1px solid ${alpha("accent2", 0.2)};
+  border-radius: 0.5rem;
+`;
+
 export const FieldMappingContainer = styled.div`
   display: flex;
   align-items: center;
 `;
 
+export interface FieldSelectButtonProps {
```

Comment:
Do we need `export` here?

Reply:
Strictly speaking, no, we don't need. But it seems like a good idea to me to export props interfaces for exported components.